src/styles/Header/Header.module.css

```
@@ -0,0 +1,17 @@
+.header {
+    height: 70px;
+    width: 100%;
+    padding: 0 2rem;
+    border-bottom: 1px solid var(--accent-color);
+    display: flex;
+    justify-content: center;
+    position: fixed;
+    top: 0;
```

```suggestion
    top: 0;
    z-index: 10;
```
I'd better add z-index at here.

---

I added the z-index here